src/components/org/PositionCard/components/PositionIconPhoto.tsx

```diff
@@ -37,13 +39,22 @@ const PositionPhotoIcon: FC<PositionPhotoIconProps> = ({
     anonymize,
     inline,
     highlightTaskOwner,
+		showPmt,
 }) => {
+
+    const isRotating = rotationInstances.length > 0;
+    const isTaskOwner = position.isTaskOwner;
+    const isPmt = position.isProjectManagementTeam && showPmt;
+
+		//show pmt as warning if allowExternalProjectManagementTeam is not enabled but non employee is assigned regardless
+		const pmtExternalWarning = isPmt && !(position.properties as Record<string, unknown>).allowExternalProjectManagementTeam && currentInstance.assignedPerson && currentInstance.assignedPerson?.accountType !== 'Employee'
```

Comment:
Indentation. 
if type `position.properties` are used other places consider extending the type with `allowExternalProjectManagementTeam`.

Reply:
Will consider, only used in one place in org admin. Needs to be done in fusion-api package